e2e/test/scenarios/embedding/embedding-snippets.cy.spec.js

```diff
@@ -115,20 +115,6 @@ features.forEach(feature => {
 
         cy.findByRole("tab", { name: "Look and Feel" }).click();
 
-        // set transparent background metabase#23477
```

Comment:
This is a block for questions. We don't offer this option for questions. [See the spec here](https://www.notion.so/metabase/Re-design-transparent-theme-for-static-embedding-and-public-links-a28e98a6e5c64301a48387c7b333d569?pvs=4#2bb0cb2ade334707a0448de8500cc75a).